.github/workflows/test_gen.yaml

```diff
@@ -109,7 +109,9 @@ jobs:
         name: hive-udf
         path: featurebyte/sql/spark/
     - name: Run tests
-      run: task test-unit
+      run: |-
+        task test-unit
+        task generate-unit-test-fixtures
```

Comment:
Run the `generate-unit-test-fixtures` after `test-unit` so that the newly generated fixtures won't affect the existing unit tests.